com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -305,13 +306,14 @@ internal static void PerformDilation()
                 for (int iterations = 0; iterations < dilationSettings.dilationIterations; ++iterations)
                 {
                     // Make sure all is loaded before performing dilation.
-                    ProbeReferenceVolume.instance.PerformPendingOperations(loadAllCells: true);
+                    ProbeReferenceVolume.instance.PerformPendingOperations();
```

Comment:
We need to make sure all cells are loaded in before performing dilation. 

If they don't all fit in memory we should split in multiple dilation passes, but data needs to be forced in, otherwise it won't work.

Reply:
Good point. I'll take a look on how to do that properly. Did not think of that.